src/generator/box_helper.cr

```diff
@@ -50,6 +50,18 @@ module Generator
       end
     end
 
+    def generate_lib_types(io : IO, callable : CallableInfo)
+      is_signal = callable.is_a?(SignalInfo)
+
+      callable.args.each do |arg|
+        # If arg_type is Void, it's probably a struct, GObjIntrospection doesn't inform that signal args are pointer when
+        # they are structs
+        arg_type = to_lib_type(arg.type_info, structs_as_void: true)
+        arg_type = "Pointer(#{arg_type})" if is_signal && arg_type == "Void"
+        io << arg_type << ", "
+      end
```

Comment:
Small change to avoid trailing commas:

```suggestion
      callable.args.each_with_index do |arg, i|
        # If arg_type is Void, it's probably a struct, GObjIntrospection doesn't inform that signal args are pointer when
        # they are structs
        arg_type = to_lib_type(arg.type_info, structs_as_void: true)
        arg_type = "Pointer(#{arg_type})" if is_signal && arg_type == "Void"

        io << ", " unless i.zero?
        io << arg_type
      end
```

Reply:
1. The code gets formatted by `crystal tool format` anyway - there are tons of trailing commas everywhere inside the generator
2. The current code requires a trailing comma because the return type is appended afterwards, so there is actually no trailing comma. The code would break with this suggested change:
   ```crystal
   Proc(Pointer(Void), <% generate_lib_types(io, vfunc) %><%= return_type %>)
   ```

Reply:
Yes, better have a simpler generator code generating trailing commas that are removed by crystal format rather than write extra logic to avoid such commas, BTW I love the fact that Crystal allows these extra commas without raising a syntax error.